src/main/java/team/gif/robot/commands/autoaim/LimelightAutoAim.java

```diff
@@ -28,67 +28,84 @@ public void initialize() {
         Drivetrain.rightTalon1.enableCurrentLimit(false);
         Drivetrain.rightTalon2.enableCurrentLimit(false);
 
-        targetLocked = false;
+        Robot.hood.setHoodDown();
```

Comment:
We don't know if the hood should be down or up. Leve that for later. Delete this line.
```suggestion
        Robot.hood.setHoodDown();
```
```suggestion

```